collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/BitmovinSdkAdapter.java

```diff
@@ -509,10 +541,25 @@ public void onReady(ReadyEvent readyEvent) {
 
             if (bitmovinPlayer.isPlaying()) {
                 stateMachine.transitionState(PlayerState.PLAYING, getPosition());
+                // autoplay
+                isVideoAttemptedPlay = true;
             } else {
                 stateMachine.transitionState(PlayerState.PAUSE, getPosition());
             }
 
         }
     };
+
+    private CountDownTimer videoStartTimeout = new CountDownTimer(Util.VIDEOSTART_TIMEOUT, 1000) {
+        @Override
+        public void onTick(long millisUntilFinished) {
+        }
+
+        @Override
+        public void onFinish() {
+            Log.d(TAG, "VideoStartTimeout finish");
```

Comment:
@sabe25 In the iOS collector the timer stops when the app is put into background, right?

Reply:
Yes it pauses the timer when it goes into background and continue when it comes back into foreground/focus
For android there is the onPause event
https://developer.android.com/reference/android/app/Activity#onPause()

Reply:
I think it's the same thing as with onDestroy event. These events are related to the main app, and track only if entire app goes into specific state. I could maybe add some new method logic in `BitmovinAnalytics` file, but then it would depend on calls form "outside" to stop timer. This would also might require moving timer logic from `BitmovinSdkAdapter` to `BitmovinAnalytics` and I'm not sure if that is good option because `BitmovinAnalytics` should be only responsible for crating and sending event data (at least as I see it).

Reply:
I see. Thanks for elaborating the problem.